apps/postybirb-ui/src/website-components/furaffinity/furaffinity-form-fields.tsx

```diff
@@ -0,0 +1,7 @@
+declare module '@postybirb/types' {
+  interface FieldTranslations {
+    species: true;
+  }
+}
+
+export {};
```

Comment:
Example of expanding FieldTranslations

Note export {} in the end of the file, it is needed to make this file run as a module and not as .d.ts file that will override the whole '@postybirb/types' typings. We love typescript, don't we?